### ISSUE TITLE
typo in pyenv-sh-deactivate? (for fish users)

### ISSUE THREAD

**Issue by slebedev** [yyuu/pyenv-virtualenv#136]
When I cd to my project that has it's  own virtualenv (or if I activate virtualenv manually), I have the following error (I use fish shell, as specified in title);

```shell
fish@localhost:~> cd myVirtualEnv
pyenv-virtualenv: deactivate
pyenv-virtualenv: activate py3.5virtenv
pyenv-virtualenv: prompt changing not work for fish.
functions: Unknown option '-g'
/var/folders/bq/6hqs9m7d75xfdhmdbbx4lvzh0000gn/T//.psub.YtAIlHrNsn (line 11): if functions -g deactivate;
                                                                                 ^
from sourcing file /var/folders/bq/6hqs9m7d75xfdhmdbbx4lvzh0000gn/T//.psub.YtAIlHrNsn
        called on line 45 of file ~/.config/fish/config.fish

in function 'pyenv'
        called on line 57 of file ~/.config/fish/config.fish
        with parameter list 'activate --quiet'

in function '_pyenv_virtualenv_hook'
        called on standard input

in event handler: handler for generic event 'fish_prompt'


       functions ‐‐ print or erase functions

   Synopsis
       functions [ ‐a | ‐‐all ] [ ‐n | ‐‐names ]
       functions ‐c OLDNAME NEWNAME
       functions ‐d DESCRIPTION FUNCTION
       functions [ ‐e | ‐q ] FUNCTIONS...

functions: Type 'help functions' for related documentation
```

As you can see in synopsis of `functions`, there are no '-g' option, but there is '-q' option, which means to test if specified function exists.

**Comment by yyuu** [comment]
Ah, you're right. I'll prepare a patch for it soon. Please give it a try once it pushed.

**Comment by yyuu** [comment]
I confirmed that \ 13b7dc172ab809c3300ee0fa777a20b900802661 works fine with `fish`. I'll create a new release including it later.

@slebedev Please ping me back if you still have any problem with `fish`....

**Comment by slebedev** [comment]
@yyuu it's all ok. I've tested it before and prepared a patch, but you fixed it already. Thanks!

**Comment by yyuu** [comment]
@slebedev Thanks for confirmation! I'll push a release to Homebrew.

**Comment by yyuu** [comment]
Released 20151229 including this.
